src/main/java/org/gridsuite/securityanalysis/server/service/SecurityAnalysisRunContext.java

```diff
@@ -65,19 +65,25 @@ public SecurityAnalysisRunContext(UUID networkUuid, String variantId, List<Strin
         this.reportType = reportType;
     }
 
-    private static SecurityAnalysisParameters buildParameters(SecurityAnalysisParametersInfos parameters, String provider) {
-        SecurityAnalysisParameters params = parameters == null || parameters.getParameters() == null ?
-                SecurityAnalysisParameters.load() : parameters.getParameters();
-        if (parameters == null || parameters.getLoadFlowSpecificParameters() == null || parameters.getLoadFlowSpecificParameters().isEmpty()) {
-            return params; // no specific LF params
+    private static SecurityAnalysisParameters buildParameters(SecurityAnalysisParameters securityAnalysisParameters,
+                                                              LoadFlowParametersInfos loadFlowParametersInfos,
+                                                              String provider) {
+        if (loadFlowParametersInfos == null || loadFlowParametersInfos.getCommonParameters() == null) {
```

Comment:
I discussed about this topic with @SlimaneAmar if I remember well, and we concluded that it can and should not be null
https://github.com/gridsuite/sensitivity-analysis-server/pull/59/files#diff-062a64e40d4b0337c144e78226db03a232e7c7aa2ecd7c5da58a65c2630e86d0R70